packages/flutter/test/cupertino/checkbox_test.dart

```diff
@@ -461,9 +467,284 @@ void main() {
       paints
         ..drrect(
           color: const Color(0xfff44336),
-          outer: RRect.fromLTRBR(13.0, 13.0, 31.0, 31.0, const Radius.circular(5)),
-          inner: RRect.fromLTRBR(17.0, 17.0, 27.0, 27.0, const Radius.circular(1)),
+          outer: RRect.fromLTRBR(15.0, 15.0, 29.0, 29.0, const Radius.circular(5)),
+          inner: RRect.fromLTRBR(19.0, 19.0, 25.0, 25.0, const Radius.circular(1)),
+        ),
+    );
+  });
+
+  testWidgets('Checkbox default colors, and size in light mode', (WidgetTester tester) async {
+    Widget buildCheckbox({bool value = true}) {
+      return CupertinoApp(
+        home: Center(
+          child: CupertinoCheckbox(
+            value: value,
+            onChanged: (bool? newValue) {},
+          ),
```

Comment:
I've noticed that your golden files have size of 800x600 despite capturing the checkbox alone. This is because `matchesGoldenFile` selects the nearest repaint boundary. 
```suggestion
          child: RepaintBoundary(
            child: CupertinoCheckbox(
              value: value,
              onChanged: (bool? newValue) {},
            ),
          ),
```

Also, the `buildCheckbox`s are pretty similar. Do you think it'd be a good idea to extract them to a stateless widget?

Reply:
That's true, they do look extremely similar. I did favor duplicating them in each test just because I thought it would be easier for someone reading the test to see everything going on within the test at a glance instead of having to navigate elsewhere, considering it's also only just a few lines of code.

Reply:
That makes sense too :)